packages/cli/src/oclif/commands/logs.js

```diff
@@ -0,0 +1,119 @@
+const BaseCommand = require('../ZapierBaseCommand');
+const { flags } = require('@oclif/command');
+const { buildFlags } = require('../buildFlags');
+const { grey } = require('colors/safe');
+const { pick } = require('lodash');
+
+const { listLogs } = require('../../utils/api');
+
+// pulled out so we can pull these explicitly to send to the server
+const commandFlags = {
+  version: flags.string({
+    char: 'v',
+    description: 'Filter logs to the specified version.'
+  }),
+  status: flags.string({
+    char: 's',
+    description: 'Filter logs to only see errors or successes',
+    options: ['any', 'success', 'error'],
+    default: 'any' // this doesn't really need to be a status
+  }),
+  type: flags.string({
+    char: 't',
+    description: 'See logs of the specified type',
+    options: ['console', 'bundle', 'http'],
+    default: 'console'
+  }),
+  detailed: flags.boolean({
+    // no char since it conflicts with --debug
+    description: 'See extra info, like request/response body and headers.'
+  }),
+  user: flags.string({
+    char: 'u',
+    description: 'Only show logs for this user. Defaults to your account.',
+    default: 'me'
+  }),
+  limit: flags.integer({
+    description:
+      'Cap the number of logs returned. Max is 50 (also the default)',
+    default: 50
+  })
+};
+
+class LogsCommand extends BaseCommand {
+  async perform() {
+    this.startSpinner('Loading logs');
+
+    const flags = pick(this.flags, Object.keys(commandFlags));
+    const { logs } = await listLogs(flags);
+    this.stopSpinner();
+    const hasLogs = Boolean(logs.length);
+
+    let headers;
+    if (this.flags.type === 'http') {
+      headers = [
+        ['Status', 'response_status_code'],
+        ['Method', 'request_method'],
+        ['URL', 'request_url'],
+        ['Querystring', 'request_params'],
+        ['Version', 'app_cli_version'],
+        ['Step ID', 'step'],
+        // ['ID', 'id'],
+        ['Timestamp', 'timestamp']
+      ];
+
+      if (this.flags.detailed) {
+        headers.concat([
+          ['Request Headers', 'request_headers'],
+          ['Request Body', 'request_data'],
+          ['Response Headers', 'response_headers'],
+          ['Response Body', 'response_content']
+        ]);
```

Comment:
When I run `zapier logs --type http --detailed`, details are not showing. The problem is `Array.concat` doesn't do an in-place update. Should change it to:

```
headers = headers.concat([ ...
```

Reply:
Ah, good catch. TS would catch this too 😁